### PR TITLE
User can see list of all scheduled messages

### PR DESCRIPTION
**Pull request by jessieay**
Design skills continue to be on fleek: 
screen shot 2015-09-18 at 11 51 59 am

### CODE REVIEW

File: spec/features/view_scheduled_messages_spec.rb [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+  private
+
+  def create_scheduled_messages
+    first_scheduled_message
+    second_scheduled_message
+  end
+
+  def first_scheduled_message
+    @first_scheduled_mesage ||= create(:scheduled_message)
```
Comment (gemfarmer) [inline spec/features/view_scheduled_messages_spec.rb]: `@first_scheduled_mesage` message is missing an 's'

Comment (jessieay) [inline spec/features/view_scheduled_messages_spec.rb]: gracias :abcd:

**Comment by jessieay** [comment]
@gemfarmer updated

**Comment by gemfarmer** [comment]
:+1: Good to merge

**Pull request merged**